examples/example_dirac/createTestingData.m

```diff
@@ -0,0 +1,26 @@
+function [ output_args ] = createTestingData( input_args )
+    %CREATETESTINGDATA This function writes some data for continuous
+    % integration tests
+    
+    !head -n -1 simulate_model_dirac.m > simulate_model_dirac_hdf.m
```

Comment:
I have never seen ! commands in matlab, is there any documentation of this feature?